packages/admin-panel-server/src/viz-builder/types.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+type AggregationObject = {
+  readonly type: string;
+  readonly config?: Record<string, unknown>;
+}
+
+type Aggregation = string | AggregationObject;
+
+type Transform = Record<string, unknown>;
+
+type DataObject = {
+  dataElements: string[];
+  dataGroups: string[];
+  aggregations: Aggregation[];
+  transform: Transform[];
+};
```

Comment:
We really do need a common way of defining types across packages... I think we have aggregation objects defined in like 3 packages now 😝 

In a refactor I'm working on I've moved the 'DbModel' types from entity-server into server-boilerplate. I may do the same for the ReportModel types from report-server and then use them here too!